Update the installation guide (#105)
glotzerlab/signac-docs: docs/source/packages.rst

    :alt: PyPI signac

The *core* **signac** package implements a simple, serverless, distributed database directly on the file system.

> Its features include:

Reviewer: ```suggestion
The **signac** package allows users to:
```

reframing with focus on what the users wants

- Manage project data with a well-defined indexable storage layout for data and metadata.
- Search and manipulate the data in existing **signac** projects.

> - Provide the data management functionality for **signac-flow** and **signac-dashboard**.

Reviewer: ```suggestion
- Unlock the use of **signac-flow** and **signac-dashboard** for additional utility.
```
Phrase the dependency as what the user gets out of it

Author: I'm not sure if unlock is the right word here, it might confuse the users that the core signac package is dependent on signac-flow and signac-dashboard. What do you think about this?

Reviewer: I want to show that signac-flow and dashboard depend on signac. But I also want to keep a parallel grammatical structure to what the user can do. Maybe enable instead of unlock?
"Enable the use of signac-flow and signac-dashboard " ?
As in: you have to use signac to enable the other packages.

**Actually, I think we can just get rid of this last bullet -- dependencies can be dealt with on the installation page** Even with these revisions it doesn't tell me much about what the user gets out of the package

Author: To show the dependency between these packages, we mention that _"You only have to install those that provide the functionality you need, however both **signac-flow** and **signac-dashboard** require the **signac** core package."_ in the introductory paragraph of `packages` page

Reviewer: Good point! I felt safe to remove that bullet then

    :target: https://pypi.org/project/signac-flow/
    :alt: PyPI signac-flow

The **signac-flow** package allows users to:

> - Implement workflows for a project data space managed with **signac**.

Reviewer 2: ```suggestion
- Implement reproducible computational workflows for a project data space managed with **signac**.
```

    :target: https://pypi.org/project/signac-flow/
    :alt: PyPI signac-flow

> The **signac-flow** package allows users to implement workflows that operate on a data space managed with **signac**:
> 
> - Implement workflows for projects managed with **signac**.

Reviewer: ```suggestion
The **signac-flow** package allows users to:

- Implement workflows for a project data space managed with **signac**.
```

trying to remove redundancy

Reviewer: added a new commit to fix first bullet redundancy

    :target: https://pypi.org/project/signac-dashboard/
    :alt: PyPI signac-dashboard

> The **signac-dashboard** package allows users to browse **signac**-managed data spaces through a web-based GUI:

Reviewer: ```suggestion
The **signac-dashboard** package allows users to browse **signac**-managed data spaces:
```
removed bc redundant in bullet
